gi-gtk-declarative/default.nix

```diff
@@ -1,13 +1,10 @@
-{ pkgs ? import <nixpkgs> {}, compiler ? "ghc865"
+{ pkgs ? import ./nixpkgs.nix, compiler ? "ghc883"
 , doCheck ? false, doBenchmark ? false
 }:
 
 let
   haskellPackages = pkgs.haskell.packages.${compiler}.override {
```

Comment:
Couldn't this just be `haskellPackages = pkgs.haskell.packages.${compiler};` (since the override no longer does anything) ?

Reply:
Yes. You are right. I kept it for future overrides, but that's perfectly fine to remove it.

Reply:
Done.